contracts/sawtooth-pike/pike.yaml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 name: pike
-version: '1.0'
+version: '1.0.0'
```

Comment:
This will potentially break existing pike clients.

Reply:
Removed the semver enforcement at this level